### ISSUE TITLE
JSON-LD "@context" needs another "@context"

### ISSUE THREAD

**Issue by gaurav** [OnroerendErfgoed/atramhasis#794]
Steps to reproduce:
1. Go to any concept, e.g. https://thesaurus.onroerenderfgoed.be/conceptschemes/ERFGOEDTYPES/c/885
2. Click "JSON/LD" to download the JSON-LD file as `885.jsonld`.
3. Run `npx jsonld-cli format -q 885.jsonld` to convert the JSON-LD file into N-Quads.

Found:
* No output.

Expected:
* N-Quads output.

Fix:
1. Download https://thesaurus.onroerenderfgoed.be/jsonld/context/skos and save it as `skos.json`, e.g. `wget "https://thesaurus.onroerenderfgoed.be/jsonld/context/skos" -O skos.json`
2. Edit skos.json by adding an additional `{"@context":{...}}` around the entire contents of the file ([as in this file](https://github.com/OnroerendErfgoed/atramhasis/files/10887096/skos.json.txt)) (I got this idea from this StackExchange answer: https://stackoverflow.com/a/61231984/27310) 
3. In 885.jsonld, change the "@context" from "https://thesaurus.onroerenderfgoed.be/jsonld/context/skos" to "./skos.json" ([as in this example](https://github.com/OnroerendErfgoed/atramhasis/files/10887106/885.jsonld.txt)).
4. Run `npx jsonld-cli format -q 885.jsonld`. Valid N-quads are now produced.

So: I think you need to change the code that generates `/jsonld/context/skos` so that it adds `{"@context":{...}}` around the current output for it to be a functional JSON-LD context.

**Comment by koenedaele** [comment]
Yes. Good catch. Thank you!

**Comment by koenedaele** [comment]
Will be fixed in pyramid_skosprovider 1.2.0. By updating the version of pyramid_skosprovider in Atramhasis demo, the following now works:

```
koen@iskierka:~/Projecten/python/skos/atramhasis_demo$ wget http://localhost:6543/conceptschemes/STYLES/c/116.jsonld
--2023-03-08 12:12:39--  http://localhost:6543/conceptschemes/STYLES/c/116.jsonld
Herleiden van localhost (localhost)... 127.0.0.1
Verbinding maken met localhost (localhost)|127.0.0.1|:6543... verbonden.
HTTP-verzoek is verzonden; wachten op antwoord... 200 OK
Lengte: 1703 (1,7K) [application/ld+json]
Wordt opgeslagen als: ‘116.jsonld’

116.jsonld                   100%[==============================================>]   1,66K  --.-KB/s    in 0s      

2023-03-08 12:12:39 (471 MB/s) - '‘116.jsonld’' opgeslagen [1703/1703]

koen@iskierka:~/Projecten/python/skos/atramhasis_demo$ npx jsonld-cli format -q 116.jsonld 
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://purl.org/dc/terms/identifier> "116"^^<http://www.w3.org/2001/XMLSchema#integer> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://purl.org/dc/terms/source> _:b0 .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://rdfs.org/ns/void#inDataset> <https://id.erfgoed.net/datasets/thesauri/stijlen_en_culturen> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#Concept> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://www.w3.org/2000/01/rdf-schema#label> "La Tène" .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://www.w3.org/2004/02/skos/core#inScheme> <https://id.erfgoed.net/thesauri/stijlen_en_culturen> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://www.w3.org/2004/02/skos/core#prefLabel> "La Tène"@nl-be .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> <http://www.w3.org/2004/02/skos/core#scopeNote> "Verwijst naar een fase en cultuur van de Europese Kelten in de late ijzertijd, genoemd naar de archeologische vindplaats La Tène, of De Ondiepe Wateren, bij het Meer van Neuchâtel in Zwitserland. De fase begon in het midden van de 5de eeuw v. Chr. en de cultuur verspreidde zich over Noord-Europa en de Britse Eilanden, om ten slotte halverwege de 1ste eeuw v. Chr. te verdwijnen. La Tène kenmerkt zich door Griekse en Etruskische invloeden op de traditionele Keltische cultuur en door de sterke geografische verspreiding van de Kelten."@nl-be .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/63> <http://purl.org/dc/terms/identifier> "63"^^<http://www.w3.org/2001/XMLSchema#integer> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/63> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#Collection> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/63> <http://www.w3.org/2000/01/rdf-schema#label> "culturen uit de metaaltijden" .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen/63> <http://www.w3.org/2004/02/skos/core#member> <https://id.erfgoed.net/thesauri/stijlen_en_culturen/116> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#ConceptScheme> .
<https://id.erfgoed.net/thesauri/stijlen_en_culturen> <http://www.w3.org/2000/01/rdf-schema#label> "Styles and Cultures" .
_:b0 <http://purl.org/dc/terms/bibliographicCitation> "1) Nederlandse Art &amp; Architecture Thesaurus, <a href=\"http://browser.aat-ned.nl/\">http://browser.aat-ned.nl/</a> \n2) VIOE"^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#HTML> .
_:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/BibliographicResource> .
```

**Comment by koenedaele** [comment]
Todo: release pyramid_skosprovider 1.2.0 and update in Atramhasis.
